openfisca_france/param/ipp_tax_and_benefit_tables_to_parameters.py

```diff
@@ -0,0 +1,1391 @@
+# -*- coding: utf-8 -*-
+
+
+import collections
+import datetime
+
+
+def fixed_bases_tax_scale(base_by_slice_name, null_rate_base = None, rates_tree = None):
```

Comment:
Is it the part that convert the IPP params in the xml files ? What is the input ?